Scalar.Common/Platforms/Mac/MacPlatform.cs

```diff
@@ -148,7 +151,17 @@ public override void IsServiceInstalledAndRunning(string name, out bool installe
 
         public override string GetTemplateHooksDirectory()
         {
-            return Path.Combine("/usr", "local", "git", ScalarConstants.InstalledGit.HookTemplateDir);
+            string gitExecPath = GitInstallation.GetInstalledGitBinPath();
+
+            // Resolve symlinks
+            string resolvedExecPath = NativeMethods.ResolveSymlink(gitExecPath);
+
+            // Compute the base installation path (../../)
```

Comment:
are we sure that git.exe will always be 3+ levels down?  I guess I'm not sure what happens here if we're too shallow.

Reply:
I think we can guarantee that, or else we won't have a place to find the non-binary assets of `git`. Perhaps the comment is a bit incorrect, since we are doing the following (with the expected dir provided as an example):

```
resolvedExecPath= "/usr/local/bin/git"
gitBinDir = "/usr/local/bin"
installBaseDir = "/usr/local"
return "/usr/local/share/git-core/templates/hooks"
```

Reply:
So if `pwd` was the same directory as `git`, then we are _really_ going up one level: `../share/git-core/templates/hooks` but we need to strip the `/git` from the initial path.

Reply:
yeah, i was going to suggest just doing a `ResolveSymlink( resolvedExecPath + "../../" + templatePath )`, (since `realpath()` will take care of all of that) but the first `../` to undo the git(.exe) filename doesn't work, so yeah maybe tweak the comment a bit.